raiden/tests/utils/blockchain.py

```diff
@@ -105,16 +120,40 @@ def geth_create_account(datadir, privkey):
     assert create.returncode == 0
 
 
-def geth_init_datadir(genesis, datadir):
-    """Initialize a clients datadir with our custom genesis block.
+def geth_bare_genesis(genesis_path, private_keys):
+    """Creates a bare genesis inside `datadir`.
+
     Args:
         datadir (str): the datadir in which the blockchain is initialized.
+
+    Returns:
+        str: The path to the genisis file.
     """
-    genesis_path = os.path.join(datadir, 'custom_genesis.json')
+    account_addresses = [
+        privatekey_to_address(key)
+        for key in set(private_keys)
+    ]
+
+    alloc = {
+        address_encoder(address): {
+            'balance': DEFAULT_BALANCE_BIN,
+        }
+        for address in account_addresses
+    }
+    genesis = GENESIS_STUB.copy()
+    genesis['alloc'] = alloc
 
     with open(genesis_path, 'w') as handler:
         json.dump(genesis, handler)
 
+
+def geth_init_datadir(datadir, genesis_path):
+    """Initialize a clients datadir with our custom genesis block.
+
+    Args:
+        datadir (str): the datadir in which the blockchain is initialized.
```

Comment:
Maybe we should try to use the sphinx style of documentation now since we want to use it to generate automatic documentation for our projects?

Check [here](http://www.sphinx-doc.org/en/1.4.8/domains.html#info-field-lists) for the format.